mmv1/third_party/terraform/services/gkehub2/resource_gke_hub_feature_test.go.erb

```diff
@@ -486,6 +487,7 @@ resource "google_gke_hub_feature" "feature" {
   location = "global"
   fleet_default_member_config {
     configmanagement {
+      version = "1.17.0"
```

Comment:
Can you change this version to a different value? So it will test if the version update is working or not.

Reply:
Done